testbed/mockdatasenders/mockdatadogagentexporter/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/testbed/mockdat
 go 1.20
 
 require (
-	github.com/DataDog/datadog-agent/pkg/trace/exportable v0.0.0-20201016145401-4646cf596b02
+	github.com/DataDog/datadog-agent/pkg/trace/exportable 8c2d00da7095
```

Comment:
`pkg/trace/exportable` was deprecated in favor of `pkg/trace`, we need to update this